package.json

```diff
@@ -72,8 +73,10 @@
     "async-mutex": "^0.4.0",
     "elliptic": "^6.5.4",
     "ethers": "^5.5.3",
+    "libxmtp": "file:../libxmtp/bindings/wasm",
     "long": "^5.2.0",
-    "siwe": "^2.1.3"
+    "siwe": "^2.1.3",
+    "xmtpv3": "file:../libxmtp/bindings/wasm"
```

Comment:
Why two dependencies on the same package?

Reply:
great catch - probably leftover from when I tried and failed to `npm link`